src/store/asana/index.ts

```diff
@@ -20,13 +20,12 @@ startWorkers();
 
 function createClient(accessToken: string, refreshToken: string) {
   const client = AsanaSdk.Client.create();
-  client.dispatcher.handleUnauthorized = () => {
+  client.dispatcher.handleUnauthorized = (async () => {
     console.error("failed to perform action - signing in again");
     store.commit("asana/clearActions");
     store.commit("asana/clearErrors");
-    store.dispatch("asana/signIn");
-    return true;
-  };
+    await store.dispatch("asana/signIn");
```

Comment:
Problem was this sign in request. Without the "await", when the access token expired, this handler would send a sign in request for each unauthorised action, leading to too many requests.